src/main/java/seedu/address/model/recipe/ServingSize.java

```diff
@@ -29,7 +30,7 @@ public ServingSize(Integer size) {
      * Checks if the size is a valid Integer value > 0.
      */
     public static boolean isValidServingSize(Integer test) {
-        return test > 0;
+        return (test > 0 && test < 1000);
```

Comment:
Same thing with this.

Reply:
idk man does it rly matter?? i think this is quite arbitrary and shouldnt make too big of a difference

Reply:
Mainly for our UG and what we tell our users the bounds are, and bounds testing. 

Example: "serving size of up to 1000" would mean 1000 is a valid entry but this check makes it such that its only valid if its <1000